src/os/bluestore/BlueFS.cc

```diff
@@ -51,6 +51,12 @@ BlueFS::BlueFS(CephContext* cct)
   discard_cb[BDEV_WAL] = wal_discard_cb;
   discard_cb[BDEV_DB] = db_discard_cb;
   discard_cb[BDEV_SLOW] = slow_discard_cb;
+  if (cct->_conf->bluefs_bdev_discard == "sync")
```

Comment:
Suggest to introduce a function to convert string name to discard_t to avoid copy-paste.

Reply:
I'll add a static function, according to your comment!